mattermost/mattermost-mobile
# [Gekidou MM-39724] - Teams Websocket Events

#### Summary
This PR handles the following teams websocket events:
* Update Team events
* Add Team Event

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-39724

#### Testing
- Open Mobile Device Simulator && connect to test server
- Open MM webapp && connect to same test server
- Remove from a Team
   - Webapp - remove youself from a Team
   - Mobile - observe team removed from team sidebar
- Add yourself to a Team
  - Webapp - add yourself to a team or create a new one
  - Mobile - observe team added to team sidebar
- Update a Team
  - Webapp - update a team (Examples: change name, add picture)
  - Mobile - observer update occures

#### Checklist


#### Device Information
This PR was tested on: 
* android emulator


#### Screenshots


#### Release Note


```release-note

```
